minard/templates/channelflagsbychannel.html

```diff
@@ -0,0 +1,102 @@
+{% extends "layout.html" %}
+{% block title %}Channel Flags By Channel{% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+<div class="page-header">
+  <h1 align="center">Channel Flags By Channel for {{run_type}} Run {{run_number}} </h1>
```

Comment:
You can print a human readable form of the run type by using the template filter run_type, i.e. `{{ run_type | run_type }}`.